Stream_support/doc/Stream_support/IOstream.txt

```diff
@@ -258,6 +258,24 @@ in those WKT types, namely:
 - `CGAL::Point_3`
 - `CGAL::Polygon_with_holes_2`
 - random access range of the above types.
+
+\section IOstreamXML Reading Adhoc XML
+
+In case you have to read data in a simple xml file with a simple structure, without existing I/O libraries
+the <a href="https://www.boost.org/doc/libs/property_tree">`boost::property_tree`</a> comes in handy.
```

Comment:
This links to a 404 error.
https://www.boost.org/doc/libs/property_tree

Reply:
I have fixed the URL in https://github.com/CGAL/cgal/pull/3931/commits/78c5ec760f982d936f2a22fc0926a921c76b3546.